extensions/binding/org.eclipse.smarthome.binding.wemo/src/main/java/org/eclipse/smarthome/binding/wemo/handler/WemoLightHandler.java

```diff
@@ -64,6 +64,8 @@
      */
     private static final int DIM_STEPSIZE = 5;
 
+    protected final static String SUBSCRIPTION = "bridge1";
```

Comment:
@hmerk Just for my understanding: Can you elaborate what "bridge1" here exactly means and whether this also works, if you have multiple ESH instances OR multiple WeMo devices? I see that this constant was used before your PR already, but it simply looks fishy to me...